docker/postgis/README.md

```diff
@@ -1,7 +1,7 @@
 # PostGIS + OSM-specific extensions Docker image
-[![](https://images.microbadger.com/badges/image/openmaptiles/postgis.svg)](https://microbadger.com/images/openmaptiles/postgis "Get your own image badge on microbadger.com") [![Docker Automated buil](https://img.shields.io/docker/automated/openmaptiles/postgis.svg)]()
+[![Docker Automated buil](https://img.shields.io/docker/automated/openmaptiles/postgis.svg)]()
```

Comment:
Typo in Docker Automated build - missing `d`

Reply:
Thanks, solved.